.github/workflows/ci.yml

```
@@ -0,0 +1,97 @@
+        uses: docker/build-push-action@676cae2f85471aeff6776463c72881ebd902dcf9
+        with:
+          context: ./ERP-BACKEND/frontend-react
+          file: ./ERP-BACKEND/frontend-react/Dockerfile
+          push: false
+          tags: erp03-frontend:ci
+
+  # Optional: build AI-BACKEND if Dockerfile exists
+  ai-build:
+    name: AI-BACKEND Build (optional)
+    runs-on: ubuntu-latest
+    if: hashFiles('AI-BACKEND/Dockerfile') != ''
```

actionlint (1.7.12)

[error] calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)